lib/smack/SmackRep.cpp

```diff
@@ -152,8 +152,13 @@ std::string SmackRep::procName(const llvm::User& U, llvm::Function* F) {
 
 std::string SmackRep::type(const llvm::Type* t) {
 
-  if (t->isFloatingPointTy())
-    return Naming::FLOAT_TYPE;
+  if (t->isFloatingPointTy()) {
+	if (!SmackOptions::BitPrecise)
```

Comment:
You are using tabs for indentation here. Change that to two spaces instead. You should never use tabs.